tululbot/utils.py

```diff
@@ -1,18 +1,38 @@
+from datetime import datetime
 import random
 
 import requests
 import yaml
 
 
+class TimeHelper:
+
+    @classmethod
+    def now():
+        return datetime.datetime.now()
```

Comment:
why is this class needed? why not simply importing `datetime` and use `.now()` directly?